### DISCUSSION TITLE
服务启动成功以后，后台上传的速率很高

### DISCUSSION THREAD

**Discussion by uninet** [sanjusss/aliyun-ddns#43] [Q&A]
![image](https://user-images.githubusercontent.com/16642055/118366079-de760180-b5d1-11eb-946c-91502f04d3f9.png)
服务启动以后，上传一直在进行，这是怎么回事？

**Comment by sanjusss** [comment]
过一段时间后还是这样？

**Reply by uninet** [reply]
![image](https://user-images.githubusercontent.com/16642055/118384886-aeb01380-b63c-11eb-9a00-0d121ebfdae6.png)
图为刚刚开启服务3分钟左右的速度变化
昨天晚上开启服务成功后半小时左右持续以2~6m/s的上传速度

**Reply by sanjusss** [reply]
有日志吗？REDO的值设置成了多少？
